criu/kerndat.c

```diff
@@ -936,8 +937,10 @@ int kerndat_has_thp_disable(void)
 	bool vma_match = false;
 
 	if (prctl(PR_SET_THP_DISABLE, 1, 0, 0, 0)) {
-		if (errno != EINVAL)
+		if (errno != EINVAL) {
+			pr_err("prctl failed.\n");	// needs review
```

Comment:
When [prctl](https://www.man7.org/linux/man-pages/man2/prctl.2.html) returns `EINVAL`, it usually has to do with invalid arguments/options. Not sure if this error message should specify that or if "prctl failed." is sufficient.

Reply:
use pr_perror instead of pr_err